bayes_opt/acquisition.py

```diff
@@ -741,7 +804,7 @@ def _copy_target_space(self, target_space: TargetSpace) -> TargetSpace:
         keys = target_space.keys
         pbounds = {key: bound for key, bound in zip(keys, target_space.bounds)}
         target_space_copy = TargetSpace(
-            None,
+            None,  # FIXME
```

Comment:
(putting this comment here because I noticed it here)

not supplying a `target_func` is something that we intentionally allow, see cell 3 [here](https://bayesian-optimization.github.io/BayesianOptimization/1.5.1/advanced-tour.html).

Reply:
It doesn't mean there's something wrong with code.

I added `FIXME` as a flag to check in future prs, since it is used in a different way than the comment below.

https://github.com/bayesian-optimization/BayesianOptimization/blob/2298f7c5c9c7b6c9b64669bacd0e87f9ba911f96/bayes_opt/target_space.py#L19-L27

Reply:
From my side, it would be okay to modify the docstrings accordingly, too. But if you would rather not handle that in this PR, that's also acceptable.

Reply:
I'll fix it soon.

Reply:
i fixed it.

Reply:
Could you make the same change for the`fun` of the `ConstraintModel`?

Reply:
I missed this one.
I've fixed it and will push it soon along with the other fixes.